templates/vhost/vhost_ssl_settings.erb

```diff
@@ -15,7 +15,9 @@
   ssl_buffer_size           <%= @ssl_buffer_size %>;
 <% end -%>
   ssl_protocols             <%= @ssl_protocols %>;
+<% if @ssl_ciphers != '' -%>
```

Comment:
I might be mistaken, but since `@ssl_ciphers` defaults to the long list of ciphers, setting it in the configuration to an empty string would work. Like so:

`ssl_ciphers   => ''`

What do you think?